app/models/member.rb

```diff
@@ -1,2 +1,12 @@
+# frozen_string_literal: true
+
+# Member Model
 class Member < ApplicationRecord
+  # SEARCH by SQL LIKE command or Algoria by config
+  def self.search_result(query)
```

Comment:
Yeah, much more readable, thanks.  Hadn't heard of citext so I appreciate the reference.   I don't know if Algolia uses case when scoring results, will find out :)